mkdocs.yml

```
@@ -11,7 +11,8 @@ nav:
   - About: index.md
   - Guide: guide.md
   - Submission Process: submission.md
-  - The Schema: schema.md
+  - The Schema: 
+      - About the Schema: schema.md
```

Maybe this will emphasize you can't avoid the schema when making a BEP because the validator needs it?

```suggestion
  - Validation: 
      - The Schema: schema.md
```